crates/holochain/tests/tests/countersigning.rs

```diff
@@ -1195,14 +1195,14 @@ async fn chc_should_respect_chain_lock() {
     let alice_rx = conductors[0].subscribe_to_app_signals("app".into());
     let bob_rx = conductors[1].subscribe_to_app_signals("app".into());
 
+    wait_for_completion(alice_rx, preflight_request.app_entry_hash.clone()).await;
+    wait_for_completion(bob_rx, preflight_request.app_entry_hash).await;
+
     // Now a commit should success because the session has finished and we shouldn't be behind the CHC.
```

Comment:
```suggestion
    // Now a commit should succeed because the session has finished and we shouldn't be behind the CHC.
```

Otherwise that's not English :)